_dev/src/store/modules/product-feed/getters.ts

```diff
@@ -69,10 +70,9 @@ export default {
     state: LocalState,
     getters,
     rootState) : Array<string> | null {
-    const targetCountriesFromLocalStorage = localStorage.getItem('productFeed-targetCountries');
-
-    if (targetCountriesFromLocalStorage) {
-      state.settings.targetCountries = JSON.parse(targetCountriesFromLocalStorage);
+    const targetCountriesFromLocalStorage = getDataFromLocalStorage('productFeed-targetCountries');
+    if (targetCountriesFromLocalStorage !== null) {
+      state.settings.targetCountries = targetCountriesFromLocalStorage;
```

Comment:
Isn't it doing twice the check ? could we not just write 
`state.settings.targetCountries = getDataFromLocalStorage('productFeed-targetCountries')`
Since the `getDataFromLocalStorage` does the check on null and retrieve empty string or array ?

Reply:
Oh yes u right, I will change it now!

Reply:
Be careful though, because some places needed to get [] if it's null. IDK if just null does the work too

Reply:
I've tried and it's not ok because we got null instead [] :/

Reply:
Not everywhere, so in those places maybe we could write 
`something = getDataFromLocalStorage('storageKey')?? []`
What do you think ?

Reply:
Yes, it's better